### ISSUE TITLE
Don't hide search in hamburger menu

### ISSUE THREAD

**Issue by probonopd** [isaacs/github#1424]
__Good:__

![ghgood](https://user-images.githubusercontent.com/2480569/49337979-0d6fa700-f614-11e8-8a34-4e69a0586aae.png)

Note there is plenty of whitespace (black space, actually) between "Explore" and the bell symbol. Nevertheless, when viewed in the browser at 100% size, we get this watered-down "responsive" design which hides the search function behind a useless mobile-phone style "hamburger" menu

__Bad:__

![ghbad](https://user-images.githubusercontent.com/2480569/49337977-09438980-f614-11e8-8cd4-2c8f3e1c632a.png)

This wastes a lot of vertical space and still does not offer the same functionality as the original. Why would one ever want to see that?! Please revert to the original (topmost) design for all screen resolutions (maybe except for mobile phones in the non-desktop view).

**Comment by xgdgsc** [comment]
I also think this sucks. and it doesn' t work with shortcut "/" when hidden. It hides search with half-tiled window in 1080p which is very common configuration.

**Comment by probonopd** [comment]
Exactly. I always have two browser windows alongside each other on a 1080p FullHD screen. I think this is a quite common configuration...
